### ISSUE TITLE
Update project readme.md with postgreSQL queries for creating admin user

### ISSUE THREAD

**Issue by Yog** [stxnext-csr/volontulo#433]
for instance like so:

```
// postgresql commands
SELECT * FROM "auth_user"
UPDATE "auth_user" SET "is_active"=TRUE;
UPDATE "auth_user" SET "is_active"=TRUE WHERE id=7;
```

there are two flags that are required to be set in two different tables. Will describe that later

This might not be necessary when task https://github.com/stxnext-csr/volontulo/issues/428 is completed earlier

**Comment by magul** [comment]
Even if #428 will be completed information about how admin is created should be visible somewhere in readme.

**Comment by magul** [comment]
I'm closing that one, because admin will be create using fixture. It will be implemented by @tomaszwozniak in PR #479.
